HandmadeMath.h

```diff
@@ -571,18 +640,8 @@ HMM_INLINE float HMM_PREFIX(PowerF)(float Base, float Exponent)
  * Utility functions
  */
 
-COVERAGE(HMM_ToRadians, 1)
-HMM_INLINE float HMM_PREFIX(ToRadians)(float Degrees)
-{
-    ASSERT_COVERED(HMM_ToRadians);
-
-    float Result = Degrees * (HMM_PI32 / 180.0f);
-
-    return (Result);
-}
-
 COVERAGE(HMM_Lerp, 1)
-HMM_INLINE float HMM_PREFIX(Lerp)(float A, float Time, float B)
+HMM_INLINE float HMM_Lerp(float A, float Time, float B)
```

Comment:
Let's add vector lerp functions - `HMM_LerpV2` etc., plus C++ overloads for `HMM_Lerp`.